converter.js

```diff
@@ -2,6 +2,7 @@
 
 import Fontmin from 'fontmin';
 import rename from 'gulp-rename';
+import fs from 'fs'
 import path from 'path';
 
 export function convertFont(inputPath) {
```

Comment:
In the future we could go fully asynchronous and move this whole thing to an async/await scheme, but I think we're good for now